CHANGELOG.md

```diff
@@ -3,7 +3,82 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.43.0
+
+This release contains a number of fixes and enhancements.
+
+### Tooling, SDK, and Runtime
+
+- Add line count to test coverage report (#4843) (authored by @FarisR99)
```

Comment:
Do you need to add link to #4843 and the rest? Are they auto-rendered ?

Reply:
I don't know. Those are just what the `make release-patch` script coughed up. I'll take a look at what we've done for previous releases, and alter as needed.

Reply:
Edits are up, and I have stripped out those auto-generated `(#...)` entries.